<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="5" hostname="AA-PC" name="framework_testng.Testng_Method_Dependencies" tests="8" failures="1" timestamp="2020-01-20T10:15:17 IST" time="0.010" errors="0">
  <testcase name="Browser_launch" time="0.008" classname="framework_testng.Testng_Method_Dependencies"/>
  <testcase name="App_Load" time="0.001" classname="framework_testng.Testng_Method_Dependencies"/>
  <testcase name="User_login" time="0.001" classname="framework_testng.Testng_Method_Dependencies">
    <failure type="java.lang.AssertionError" message="expected [email] but found [mail]">
      <![CDATA[java.lang.AssertionError: expected [email] but found [mail]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at framework_testng.Testng_Method_Dependencies.User_login(Testng_Method_Dependencies.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- User_login -->
  <testcase name="ComposeEmail" time="0.000" classname="framework_testng.Testng_Method_Dependencies">
    <skipped/>
  </testcase> <!-- ComposeEmail -->
  <testcase name="Search_email" time="0.000" classname="framework_testng.Testng_Method_Dependencies">
    <skipped/>
  </testcase> <!-- Search_email -->
  <testcase name="Verify_Email_At_inbox" time="0.000" classname="framework_testng.Testng_Method_Dependencies">
    <skipped/>
  </testcase> <!-- Verify_Email_At_inbox -->
  <testcase name="Verify_Email_At_Sentemails_AfterSendEmail" time="0.000" classname="framework_testng.Testng_Method_Dependencies">
    <skipped/>
  </testcase> <!-- Verify_Email_At_Sentemails_AfterSendEmail -->
  <testcase name="User_Signout" time="0.000" classname="framework_testng.Testng_Method_Dependencies">
    <skipped/>
  </testcase> <!-- User_Signout -->
</testsuite> <!-- framework_testng.Testng_Method_Dependencies -->
